src/entity/Source.ts

```diff
@@ -111,8 +115,9 @@ export class Source {
 
   @Column({ type: 'jsonb', default: {} })
   @Index('IDX_source_flags_featured', { synchronize: false })
+  @Index('IDX_source_flags_posts_members_threshold', { synchronize: false })
```

Comment:
@idoshamun the index had to be declared here. Unlike the other composite indexes we have, we will use jsonb columns on this one, and the top-level index does not accept `synchronize: false` value.